cluster/examples/kubernetes/ceph/object.yaml

```diff
@@ -74,6 +74,8 @@ spec:
     annotations:
     #  key: value
     # A key/value list of labels
+      #service.beta.openshift.io/serving-cert-secret-name: <name of tls secret yet to generate>
```

Comment:
If this is only an openshift annotation, let's revert this file and just use the example in object-openshift.yaml